turtlebot_actions/launch/server_turtlebot_move_vel_ctrl.launch

```diff
@@ -0,0 +1,11 @@
+<launch>
+  <!-- Keep things as accurate as possible, don't use a smoother -->
+  <!-- <include file="$(find turtlebot_actions)/launch/includes/velocity_smoother.launch.xml"/>  -->
```

Comment:
I wouldn't recommend disabling the smoother. It make the TurtleBot look a lot better, and also improves overall performance for tracking and localization. Without the smoother it tends to bound around at very high accelerations.